config/dev1/sys.config

```diff
@@ -19,7 +19,8 @@
       {db_path, "."},
       {persist, false},
       {autostart, false},
-      {aec_pow_cuckoo, {"lean16", "-t 5", 16}}
+      {aec_pow_cuckoo, {"lean16", "-t 5", 16}},
+      {expected_mine_rate, 100}
```

Comment:
I see [default is 5 minutes per block](https://github.com/aeternity/epoch/pull/450/files#diff-312e5e881622101a1720cef694345ee9R11), this is 100 milliseconds per block. Ok.